ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -565,3 +512,35 @@ def check_capacity_breakdown(self, project_name, pod_name):
             logger.error(f"The pod {pod_name} not found on capacity_breakdown")
             res = False
         return res
+
+    def get_blockpools_compression_status_from_storagesystem(self) -> tuple:
+        """
+        Initial page - Data Foundation / Storage Systems tab / StorageSystem details / ocs-storagecluster-cephblockpool
+        Get compression status from storagesystem details and ocs-storagecluster-cephblockpool
+
+        Returns:
+            tuple: String representation of 'Compression status' from StorageSystem details page and
+            String representation of 'Compression status' from ocs-storagecluster-cephblockpool page
+
+        """
+
+        self.odf_console_plugin_check()
+        self.navigate_cephblockpool()
+
+        logger.info(
+            f"Get the 'Compression status' of '{constants.DEFAULT_CEPHBLOCKPOOL}'"
+        )
+        compression_status_blockpools_tab = self.get_element_text(
+            self.validation_loc["storagesystem-details-compress-state"]
+        )
+        logger.info(
+            f"Click on '{constants.DEFAULT_CEPHBLOCKPOOL}' link under BlockPools tab"
+        )
+        self.do_click(
+            self.validation_loc[f"{constants.DEFAULT_CEPHBLOCKPOOL}"],
```

Comment:
the type of {constants.DEFAULT_CEPHBLOCKPOOL} is set?
```
a={1}
a
{1}
type(a)
<class 'set'>```

Reply:
there are quotes and a leading "f" before quotes to use constant string var constants.DEFAULT_CEPHBLOCKPOOL

Reply:
yeap, your're right